terraform/aws/modules/compute/main.tf

```diff
@@ -104,29 +104,42 @@ resource "aws_ecs_service" "km_ecs_service" {
     container_port   = 80
   }
   network_configuration {
-    assign_public_ip = true
+    assign_public_ip = false
     subnets          = var.private_subnet
-    security_groups  = [ var.elb_sg ]
+    security_groups  = [var.elb_sg]
   }
   tags = merge(var.default_tags, {
   })
 }
 
 resource "aws_cloudwatch_log_group" "km_log_group" {
   name              = "km_log_group_${var.environment}"
-  retention_in_days = 1
+  retention_in_days = 120
```

Comment:
**Tenable.cs remediation**
 
**[HIGH]: Ensure log retention period of at least 90 days retention period for AWS CloudWatch Log Group**
 
**Vulnerability**
AWS CloudWatch Log Group does not have a log retention period of at least 90 days. It is recommended that logs expire after 90 days.
 
**Remediation**
AWS CloudWatch Log Group can be set with a log retention policy using AWS Console.
 In AWS Console - 
 1. Sign in to AWS Console and go to the CloudWatch console.
 2. Select Log Groups in the navigation pane.
 3. Select the log group to update.
 4. Select 'Edit retention' and change the log retention value to 90 days.

source: tenablecs-